docs/contributing-guide.md

```diff
@@ -0,0 +1,90 @@
+# Contributing guide
```

Comment:
Great !
To be usable directly on github, We should put in in a `CONTRIBUTING.md` at the project root and reference it directly with in mkdocs.yml